src/OrleansAzureUtils/OrleansAzureUtils.csproj

```diff
@@ -114,25 +114,11 @@
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- Begin Orleans: Without these lines the project won't build properly -->
```

Comment:
This change should be un-reverted so that the code generator can operate at build-time. The runtime code generator cannot access those internal types.